src/cmd/completion.go

```diff
@@ -62,7 +62,7 @@ func completionCommands(cmd *cobra.Command, _ []string, _ string) ([]string, cob
 	commandNames := []string{}
 	commands := cmd.Root().Commands()
 	for _, command := range commands {
-		if strings.Contains(command.Name(), "complet") {
+		if strings.Contains(command.Name(), "complete") {
```

Comment:
I wonder if there was a deliberate desire to search for a sub-string or check the prefix here?  It's curious that it uses `strings.Contains` instead of a simpler comparison.